doc/Command_Index_User.tex

```diff
@@ -1885,25 +1857,316 @@ \subsection{Distances}\label{distances}
+
+The minimum value of \texttt{additionaltopspacethreshold} is $-1$ (additional top space for anything above the top staff line) and the maximum value is currently $6$ (no additional top space).
+
+Note that if there are high notes, increasing \texttt{spaceabovelines} may not add any visible space above the notes. To ensure a minimum space above the notes, set \texttt{spaceabovelines} to a length greater than $(1+\texttt{additionaltopspacethreshold})$. That is, with all other settings at their defaults, \texttt{spaceabovelines} should be greater than $3~\textrm{steps} \approx 0.41~\textrm{cm}$.
+
+\begin{quote}
+\verb|\grechangedim{spaceabovelines}{0.5cm}{scalable}|
+\grechangedim{spaceabovelines}{0.5cm}{scalable}
+
+\begin{center}
+\rowcolors{1}{white}{white}%
+\begin{tabular}{ccccc}
+  \lines{\gabcsnippet{i[0 steps](i)}} &
+  \lines{\gabcsnippet{j[0 steps](j)}} &
+  \lines{\gabcsnippet{k[0 steps](k)}} &
+  \lines{\gabcsnippet{l[0 steps](l)}} &
+  \lines{\gabcsnippet{m[1 step](m)}}
+\end{tabular}
+\end{center}
+\end{quote}
+
+\begin{minipage}{\linewidth}
+High notes also cause the above-lines text, if any, to be raised. The above-lines text has its own threshold, \texttt{additionaltopspacealtthreshold} (default 0):
+\begin{center}
+\rowcolors{1}{white}{white}%
+\begin{tabular}{ccccc}
+  \lines{\gabcsnippet{<alt>alt</alt>i[0 steps](i)}} &
+  \lines{\gabcsnippet{<alt>alt</alt>j[0 steps](j)}} &
+  \lines{\gabcsnippet{<alt>alt</alt>k[1 step](k)}} &
+  \lines{\gabcsnippet{<alt>alt</alt>l[2 steps](l)}} &
+  \lines{\gabcsnippet{<alt>alt</alt>m[3 steps](m)}}
+\end{tabular}
+\end{center}
+\end{minipage}
+
+Because the above-lines text and the topline have different thresholds, they may get raised by different amounts. The two preceding examples combine to look like this:
+\begin{quote}
+\verb|\grechangedim{spaceabovelines}{0.5cm}{scalable}|
+\grechangedim{spaceabovelines}{0.5cm}{scalable}
+  
+\begin{center}
+\rowcolors{1}{white}{white}%
+\begin{tabular}{ccccc}
+  \lines{\gabcsnippet{<alt>alt</alt>i(i)}} &
+  \lines{\gabcsnippet{<alt>alt</alt>j(j)}} &
+  \lines{\gabcsnippet{<alt>alt</alt>k(k)}} &
+  \lines{\gabcsnippet{<alt>alt</alt>l(l)}} &
+  \lines{\gabcsnippet{<alt>alt</alt>m(m)}}
+\end{tabular}
+\end{center}
+\end{quote}
+
+NABC neumes use a different threshold, \texttt{additionaltopspacenabcthreshold}. This defaults to $4$, meaning that only the very highest notes affect the positioning of NABC neumes.
+\begin{quote}
+\grechangedim{abovelinestextheight}{0.1cm}{scalable}%
+\verb|\grechangedim{abovelinestextheight}{0.1cm}{scalable}| \\
+\grechangedim{abovelinestextraise}{0.8cm}{scalable}%
+\verb|\grechangedim{abovelinestextraise}{0.8cm}{scalable}|%
+\gresetlyriccentering{syllable}%
+\begin{center}
+\rowcolors{1}{white}{white}%
+\begin{tabular}{ccccc}
+  \lines{\nabcsnippet{m|vi[0 steps](m|vi)}} &
+  \lines{\nabcsnippet{m_1|vi[0 steps](m_1|vi)}} &
+  \lines{\nabcsnippet{m'1|vi[1 step](m'1|vi)}} &
```

Comment:
The straight quote in the lyrics of this and the next snippet are being converted to curly quotes in the output.  Please make use of `\textsinglequote` in the lyrics to get the quote to print correctly.

Reply:
Done (and changed font to match similar examples elsewhere)